File: Data Structures & Algorithms/concatenation-of-array/submission-1.cs
public class Solution {
    public int[] GetConcatenation(int[] nums) {
        int n = nums.Length;
        int[] output = new int [2 * n];
        for (int i = 0; i < n; i++) {
            output[i] = output[i + n] = nums[i];
        }
        return output;
    }
}

File: Data Structures & Algorithms/remove-element/submission-1.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        nextPosition = 0;

        for current in range(len(nums)):
            if nums[current] != val:
                nums[nextPosition] = nums[current]
                nextPosition += 1
            
        return nextPosition